web/src/app/modules/shared/components/form-view-container/form-view-container.component.html

```diff
@@ -1,111 +1,108 @@
-<div [formGroup]="formGroupContainer" *ngFor="let field of form?.fields; trackBy: trackByFn">
-  <ng-container [ngSwitch]="field.type">
 
-    <ng-container *ngSwitchCase="'checkbox'">
-      <clr-checkbox-container>
-        <label [for]="field.name">{{ field.label }}</label>
-        <clr-checkbox-wrapper
-          *ngFor="let opt of fieldChoices(field); trackBy: trackByFn"
-        >
+<cds-form-group control-width="shrink">
```

Comment:
I think layout is working as expected and only allowed for radio buttons/checkboxes. It doesn't solve the original issue though